src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+    protected String taskDescription;
+    protected boolean isDone;
+
+    public Task(String taskDescription) {
+        this.taskDescription = taskDescription;
+        this.isDone = false;
+    }
+
+    public String getTaskDescription() {
+        return taskDescription;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");
+    }
```

Comment:
Perhaps you could provide a comment regarding these two Unicodes, or define them separately. This would make it easier to understand what they are.